src/hearings/resolvers/users-info-using-codes-resolver.resolve.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+import { Resolve } from '@angular/router';
+import { select, Store } from '@ngrx/store';
+import { Observable, of } from 'rxjs';
+import { catchError, map, switchMap, take } from 'rxjs/operators';
+import { JudicialUserModel } from '../models/judicialUser.model';
+import { PanelRequirementsModel } from '../models/panelRequirements.model';
+import { JudicialRefDataService } from '../services/judicial-ref-data.service';
+import * as fromHearingStore from '../store';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UsersInfoUsingCodesResolver implements Resolve<JudicialUserModel[]> {
```

Comment:
If this can be renamed to JudicialUserSearchResolver please? And the file name judicial-user-search-resolver.ts